riscv-asm.md

```diff
@@ -183,6 +183,41 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 .variant_cc  | symbol_name                    | annotate the symbol with variant calling convention
+.attribute   | name, value                    | RISC-V object attributes, more detailed description see [.attribute](#.attribute).
+
+## <a name=.attribute></a> `.attribute`
+
+`.attribute` directive is used for record information about an object
```

Comment:
I'm not a native English speaker myself, so take this with a grain of salt, but I would probably tweak this as follows. Nits:
- "The `.attribute` directive" (add "The")
- "to record" (replace "for" with "to")
- "check for compatibility" (add "for")